src/swipe/dto/create-swipe.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsString } from 'class-validator';
+import { UserSwipeType } from '../entities';
+
+export class SwipeUserDto {
```

Comment:
naming convention
filename should be related to class name

`export class CreateSwipeDto`